Assets/NoesisGUI/Xaml/HUD.xaml

```diff
@@ -79,6 +79,19 @@
 
                 <uc:PromptGUI x:Name="PROMPT" Grid.Row="4" Grid.Column="4" Grid.ColumnSpan="2" HorizontalAlignment="Center" VerticalAlignment="Center"/>
 
+                
+                <!--<Rectangle Grid.Column="4" Grid.ColumnSpan="3" Grid.Row="5" Height="113" Width="335" Fill="#D8FF8B00" Margin="0,0,12,0">
+                    <Rectangle.OpacityMask>
+                        <ImageBrush ImageSource="bar.png"/>
+                    </Rectangle.OpacityMask>
+                </Rectangle> -->
```

Comment:
unnecessary block of comment?

Reply:
Both yes and no, it depends if we will fix the Image opacity bug.